Edits Signer Pages (#4814)
w3f/polkadot-wiki: docs/general/polkadot-vault.md

:::

> :::warning Use a smartphone that is no more than 2-3 years old
> 
> Note that although the Vault app is available for old smartphones, different versions will be
> installed according to the phone's hardware. Old phones like iPhone 6 will install Parity Signer
> (the old brand name of the Vault app) with limited capabilities. There will be no log, no warning if
> the phone was connected while not using the app, no metadata updates possible, and no option to add
> new networks. This would not be as secure as the latest version of the app. We recommend using a
> smartphone that is no more than 2-3 years old.
> 
> :::

Reviewer: The point to be conveyed here is that the smartphone should be compatible with the latest version on the app. There could be smartphones that are older that do support the latest version or some latest ones that are not compatible. 

```suggestion

:::warning Smartphone compatibility with Polkadot Vault

Note that although the Vault app is available for old smartphones, different versions will be
installed according to the phone's hardware. For example, smartphones like iPhone 6 will install Parity Signer
(the old brand name of the Vault app) with limited capabilities. There will be no log or warning if
the phone was connected to the internet while not using the app. Also, no metadata updates are possible, and no option to add
new networks. This would not be as secure as the latest version of the app. We would recommend that you use smartphones compatible with the latest Polkadot Vault app.

:::
```

## Security Notes

:::info Remove SIM cards and Forget Networks

To avoid unintended connection of your phone to the Internet, remove SIM cards, reset eSIM settings,

> and forget any WiFi network. In this way, the only possibility to have an intended internet

Reviewer: ```suggestion
and forget any added WiFi networks. In this way, the only possibility of having an unintended internet
```

> connection is either with a SIM card or a WiFi router.

Reviewer: ```suggestion
connection is either through the cellular network or through WiFi connection.
```
